entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/query/OCondition.kt

```diff
@@ -125,3 +125,16 @@ class OFieldExistsCondition(
         builder.append("not(").append(field).append(" is null)")
     }
 }
+
+class OInstanceOfCondition(
+    val instanceOf: String,
```

Comment:
Lets better rename it to 
```suggestion
    val className: String,
```